eng/SpecializedTestRunsheetBuilderBase.targets

```diff
@@ -33,9 +33,17 @@
     NOTE: The final piece of the puzzle is in eng/AfterSolutionBuild.targets, where we combine the runsheets from all the test projects into a single runsheet.
     -->
 
+  <!--
+    Skip Templates, Playground, and EndToEnd tests for specialized test workflows as these are known to be broken.
+    -->
+  <PropertyGroup>
+    <_IsUnsupportedProject>false</_IsUnsupportedProject>
+    <_IsUnsupportedProject Condition="$(MSBuildProjectName.Contains('Templates')) or $(MSBuildProjectName.Contains('Playground')) or $(MSBuildProjectName.Contains('EndToEnd'))">true</_IsUnsupportedProject>
```

Comment:
I'm assuming we still have coverage for these test projects? What is considered an specialized test run? Is that outerloop and/or quarantine?

Reply:
Outerloop, and Quarantine only. Yep, all the tests are run in the GH CI runs. Because these workflows use arcade, they need some patching to make the template/playground/e2e tests to run.

Reply:
> Because these workflows use arcade,

Assuming you meant Helix?